CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# How to Contribute
+We want you to contribute!
+
+Any and all contributions are appreciated, even if your change does not get accepted.
+
+
+## Getting Started
+Coming soon :)
+
+## Pull Requests
+  - Please squash your commits before merging.
```

Comment:
maybe provide a how-to or link? (ie "see [here](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html)")